website/layouts/_default/baseof.html

```diff
@@ -2,9 +2,10 @@
 <html lang="en">
 
 <head>
+    {{ $version     := index (split .File.Path "/") 0 }}
```

Comment:
Tiny nit: why do we need so many spaces here between the `$version` and the `:=`?

Reply:
oh...my bad
would fix up